src/OrchardCore.Modules/OrchardCore.Resources/Liquid/MetaTag.cs

```diff
@@ -23,6 +23,7 @@ public static async ValueTask<Completion> WriteToAsync(List<FilterArgument> argu
             string httpEquiv = null;
             string charset = null;
             string separator = null;
+            bool replace = false;
```

Comment:
I would rename this to bool append.

Reply:
The PR is fine though. It's just that we could use terms that we already have and just make the logic the opposite by defining the "append" as true by default.

Reply:
It seemed safer to use something that would default to `false` to keep the current code as the default (for example, when invalid values are passed to `replace`).